cpp/ql/src/Likely Bugs/Format/NonConstantFormat.ql

```diff
@@ -69,7 +69,10 @@ predicate isNonConst(DataFlow::Node node) {
   // Parameters of uncalled functions that aren't const
   exists(UncalledFunction f, Parameter p |
     f.getAParameter() = p and
-    p = node.asParameter()
+    p = node.asParameter() and 
+    // Ignore main's argv parameter as it is already considered a `FlowSource`
+    // not ignoring it will result in path redundancies
+    (f.getName() = "main" implies p.getName() != "argv")
```

Comment:
This assumes the parameter is always called `argv`. That does not have to be true.

Reply:
Good point. I should've copied more verbatim what the current argv model is doing. I'll fix that.

Reply:
Fixed